packages/sanity/src/core/studio/components/navbar/search/components/filters/documentTypes/DocumentTypesButton.tsx

```diff
@@ -0,0 +1,59 @@
+import {SelectIcon} from '@sanity/icons'
+import {Button, Flex, Popover, Text, useClickOutside} from '@sanity/ui'
+import React, {useCallback, useMemo, useState} from 'react'
+import {POPOVER_VERTICAL_MARGIN} from '../../../constants'
+import {useSearchState} from '../../../contexts/search/useSearchState'
+import {documentTypesTruncated} from '../../../utils/documentTypesTruncated'
+import {FilterPopoverWrapper} from '../common/FilterPopoverWrapper'
+import {DocumentTypesPopoverContent} from './DocumentTypesPopoverContent'
+
+export function DocumentTypesButton() {
+  const [open, setOpen] = useState(false)
+  const [buttonElement, setButtonElement] = useState<HTMLElement | null>(null)
+  const [popoverElement, setPopoverElement] = useState<HTMLElement | null>(null)
+
+  const {
+    state: {
+      fullscreen,
+      terms: {types},
+    },
+  } = useSearchState()
+
+  const handleClose = useCallback(() => setOpen(false), [])
+  const handleOpen = useCallback(() => setOpen(true), [])
+
+  useClickOutside(handleClose, [buttonElement, popoverElement])
+
+  const title = useMemo(() => documentTypesTruncated({types}), [types])
+
+  return (
+    <Popover
```

Comment:
I think it would be nice to have consistent radius on all popovers. Currently the main search popover/dialog has `radius={2}`, while other popovers have `radius={3}` (in `AddFilterButton` and `DocumentTypesButton`).

Maybe we can export a const from `constants.ts` to keep this consistent more easily?

```ts
export const POPOVER_RADIUS = 2 
```


<img width="983" alt="radius" src="https://user-images.githubusercontent.com/15094168/205930792-2e85dd42-e5e7-41fb-bbed-71354f7bc01b.png">

Reply:
I think we should use `radius={2}` as the focus outline will match well with that radius
<img width="1187" alt="rad" src="https://user-images.githubusercontent.com/15094168/205933592-4b0d2a72-d647-41f5-8221-7dc5cecccf2d.png">

Reply:
Great shout picking up this inconsistency, the smaller radius looks much nicer!